lib/bowling_game.rb

```diff
@@ -0,0 +1,112 @@
+require_relative 'scorecard'
```

Comment:
You were right that it was a little bit hard to make any meaningful comments without the challenge myself. 
I am able to follow most of code and what your methods do. I think you have followed SRP well, the two classes splitting responsibility of rolling and scoring is nice! Most of your methods are really short as well. I like how you have split it in to roll 1 and roll 2 for clarity also!